server/src/proxies.js

```diff
@@ -108,12 +108,12 @@ export default {
 
   kai: {
     pathFilter: "/kai",
-    target: KONVEYOR_ENV.TACKLE_HUB_URL || "http://localhost:9002",
+    target: KONVEYOR_ENV.KAI_URL || "http://localhost:9003",
```

Comment:
FWIW, in the current setup on the dev server, ports go like this:

:9000 -- server proxy to webpack dev server
:9001 -- port forward to keycloak
:9002 -- port forward to hub
:9003 -- webpack dev server

I swapped the server-in-dev-mode around so the base port :9000 is always used and always is handled by the express server first.